core/data/define_spec.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../spec_helper'
+require_relative 'fixtures/classes'
+
+ruby_version_is "3.2" do
+  describe "Data.define" do
+    describe "stored as constant" do
+      it "accepts no arguments" do
+        EmptyData = Data.define
+        EmptyData.members.should == []
+        EmptyData.inspect.should == "EmptyData"
+      end
+
+      it "accepts symbols" do
+        MovieWithSymbol = Data.define(:title, :year)
+        MovieWithSymbol.members.should == [:title, :year]
+        MovieWithSymbol.inspect.should == "MovieWithSymbol"
+      end
+
+      it "accepts strings" do
+        MovieWithString = Data.define("title", "year")
+        MovieWithString.members.should == [:title, :year]
+        MovieWithString.inspect.should == "MovieWithString"
+      end
+
+      it "accepts first argument as string" do
+        BlockbusterMovie = Data.define("Blockbuster", :title, :year)
+        BlockbusterMovie.members.should == [:title, :year]
+        BlockbusterMovie.inspect.should == "Data::Blockbuster"
+      end
```

Comment:
I'm not sure it's useful to test this, because it is simply the behavior of constants to name modules and classes (and visible in `Module#inspect`), nothing special about Data for that.
But we could still have a single example here to show that.
If you want to keep it, create the constant under `DataSpecs` to avoid defining a new global constant.

Reply:
I removed it. 👍